### PR TITLE
Update example endpoint

### PR DESCRIPTION
**Pull request by NatashaJIbrahim**
Hopefully this will prevent partner confusion.

### CODE REVIEW

File: _pages/production.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ sidenav:
 
 Our integration documentation includes endpoint urls for our sandbox environment <https://idp.int.identitysandbox.gov/>. Our production environment is located at <https://secure.login.gov/>. The URL path to each endpoint remains the same. Only the domain will change. For example, the authorization endpoint will change as follows:
 * OpenID Connect: <https://secure.login.gov/openid_connect/authorize>
-* SAML: <https://secure.login.gov/api/saml/auth2021>
+* SAML: <https://secure.login.gov/api/saml/auth2022>
```
Comment (zachmargolis) [inline _pages/production.md:28]: should we make a variable for these? We could do a page variable for `saml_year` or we could just do it based on the currenet calendar year?

```suggestion
* SAML: <https://secure.login.gov/api/saml/auth{{ page.saml_year }}>
```

```suggestion
* SAML: <https://secure.login.gov/api/saml/auth{{ site.time | date: "%Y" }}>
```

Comment (zachmargolis) [reply _pages/production.md:28]: (and the rest of the endpoints too)

Comment (NatashaJIbrahim) [reply _pages/production.md:28]: That would be ace! I can see if I have some bandwidth tonight to make that happen.

File: _pages/production.md [hunk 2]
```diff
@@ -19,17 +19,19 @@ sidenav:
...
 Our integration documentation includes endpoint urls for our sandbox environment <https://idp.int.identitysandbox.gov/>. Our production environment is located at <https://secure.login.gov/>. The URL path to each endpoint remains the same. Only the domain will change. For example, the authorization endpoint will change as follows:
 * OpenID Connect: <https://secure.login.gov/openid_connect/authorize>
-* SAML: <https://secure.login.gov/api/saml/auth2022>
+* SAML: <https://secure.login.gov/api/saml/auth{{ page.saml_year }}>
 
 Please be aware that the IDP certificate (X509 Certificate) in the production environment is different from the IDP certificate in the sandbox environment. The production IDP certificates can be found here:
 * OpenID Connect: <https://secure.login.gov/api/openid_connect/certs>
-* SAML: <https://secure.login.gov/api/saml/metadata2022>
+* SAML: <https://secure.login.gov/api/saml/auth{{ page.saml_year }}>
+>>>>>>> c258e36 (Use liquid for saml endpoint dates)
```
Comment (zachmargolis) [inline _pages/production.md:34]: looks like a merge conflict note slipped in 😬

Comment (NatashaJIbrahim) [reply _pages/production.md:34]: ugh - thanks for catching that!

Comment (NatashaJIbrahim) [reply _pages/production.md:34]: I should have waited until I had fresh morning eyes on it :-/

Comment (zachmargolis) [reply _pages/production.md:34]: PR to fix: https://github.com/18F/identity-dev-docs/pull/253

**Pull request merged**